### ISSUE TITLE
When a member is manually removed, unsubscribe them form the newsletter

### ISSUE THREAD

**Issue by nitro2k01** [makerspace/makeradmin#256]
Recently a member asked to end their membership and be deleted from the member database. They still continued receive the newsletter and complained. When a member is deleted manually by an admin, make sure they are also unsubscribed from the newsletter to keep the databases in sync.

**Comment by emanuelen5** [comment]
The news letter is managed by another platform called mail chimp (unrelated to makeradmin). People are added to the news letter manually by exporting the list of members from makeradmin and then importing them into mail chimp. We usually email people who have been members the last 1-2 years (i.e. who have shown up in the member export the last 1-2 years). People who are removed from makeradmin will stop showing up in the member export.

Anyway, the only way to fix this at the moment (and the path of least resistance) would be to just add it to the routine when deleting members.

**Comment by nitro2k01** [comment]
Oh, I was not aware that this is how it worked. So if I understand things correctly, new members don't start receiving the newsletter until a manual sync has been done either? This seems like a good opportunity for automation tbh.

**Comment by emanuelen5** [comment]
> Oh, I was not aware that this is how it worked. So if I understand things correctly, new members don't start receiving the newsletter until a manual sync has been done either? This seems like a good opportunity for automation tbh.

Yes, exactly. An export of members must be perfomed in makeradmin (which basically just gives a list of email addresses and names), and they are then imported manually by copy-paste into mail chimp.

**Comment by HalfVoxel** [comment]
> This seems like a good opportunity for automation tbh.

Considering that it takes maybe a minute to copy paste the member list, and that making an automation would probably take at least one or a few hours, I'm not so sure it is. Would be nice, of course, but probably not time efficient.

**Comment by emanuelen5** [comment]
Won't fix at the moment
